.gitlab/internal_image_deploy/internal_image_deploy.yml

```diff
@@ -47,6 +47,51 @@ docker_trigger_internal:
       --variable TARGET_ENV
       --variable DYNAMIC_BUILD_RENDER_TARGET_FORWARD_PARAMETERS"
 
+docker_trigger_internal-fips:
```

Comment:
Not needed for this to be merged, but we could probably share more common code between this job and the regular `docker_trigger_internal` one

Reply:
Agreed, lets make this change in a follow up PR